docs/reference/limitations/openstack.md

```diff
@@ -20,6 +20,15 @@ It also means that you cannot use the following `openstack` CLI commands; they a
   > You can use `openstack project list`, but this will only list the project(s) that your user account has access to.
 * `openstack domain <subcommand>`
 
+## Nova
+
+### Nested virtualization
+
+Running nested virtualization is only supported for Nova instances running Linux as the guest operating system.
+The guest must run a [Linux kernel](https://www.kernel.org/) of version 5.0 or later, and [QEMU/KVM](https://www.qemu.org/) 4.1 or later.
```

Comment:
Also, if I understand correctly, I believe this is more accurate:
 ```suggestion
Guests must be created with [QEMU/KVM](https://www.qemu.org/) 4.1 or later, and should run a [Linux kernel](https://www.kernel.org) of version 5.0 or later.
```

Reply:
I rephrased this slightly, and added information on what base images should be good to use for this purpose.

Reply:
Very nice!